projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "event_queue.h"
+
+#include "status.h"
+
+typedef enum { LIGHTS_BOARD_FRONT, LIGHTS_BOARD_REAR, NUM_LIGHTS_BOARDS } LightsBoard;
```

Comment:
Nit: style

Reply:
moved to `lights_config.h` and named it accordingly